### DISCUSSION TITLE
Screenshots for F-Droid

### DISCUSSION THREAD

**Discussion by mondlicht-und-sterne** [koreader/koreader#7111] [Show and tell]
You can add Screenshots for F-Droid, too, if you want. https://f-droid.org/en/docs/All_About_Descriptions_Graphics_and_Screenshots/ https://gitlab.com/snippets/1895688 That was simply information about what is still possible. You would then only have to enter the screenshots in fastlane.

**Comment by Frenzie** [comment]
I'm not entirely sure what you're saying, but we have added screenshots for F-Droid, see https://github.com/koreader/koreader/issues/2040#issuecomment-623413586 and preceding.

**Reply by pazos** [reply]
Yup. Somehow f-droid doesn't pick our fastlane metadata. I wonder if that's because we're using a submodule.
